modules/date-localizer/main/index.coffee

```diff
@@ -73,6 +73,10 @@ class DateTimeLocalizer extends PreferencesHandler
     if short then hx.zeroPad(month + 1)
     else ['Jan','Feb','Mar','Apr','May','Jun','Jul','Aug','Sep','Oct','Nov','Dec'][month]
 
+  # localise the month in the format of mmm (Jan, Feb etc.) or 01, 02 etc.
+  fullMonth: (month) ->
+    ['January','February','March','April','May','June','July','August','September','October','November','December'][month]
```

Comment:
This is one of the v2 features - The month displayed in the calendar has changed from `MMM / YYYY` to `MMMM YYYY` (e.g. from `Nov / 2018` to `November 2018`)